app/controllers/api/v1/demographics/districts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::Demographics::DistrictsController < Api::ApiController
+  respond_to :json
+
+  def show
+    district = District.find_by(slug: params["slug"])
+    year = SchoolYear.find_by(years: params["year"])
+    if year && district
+      se = StudentEnrollment.where(district_id: district.id, school_year_id: year.id)
```

Comment:
Something to consider here is to be careful of these short variable names, even when they're short lived. Sometimes it is just nice to have the whole variable in front of you (i.e., `student_enrollment`) when you're debugging at 1:30am. 😄